docs/using_eessi/cvmfs_client.md

```diff
@@ -0,0 +1,3 @@
+# Native CernVM-FS client
+
+TBD
```

Comment:
Do we want to finish this in this PR?

What do you have in mind here?

I think setting up the environment using the `source` command should be a separate page, since whether you have native access or using a container doesn't matter much from that point on.